src/components/Personalization/createAutoRenderingHandler.js

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import addRenderToExecutedDecisions from "./utils/addRenderToExecutedDecisions";
+
+export default ({
+  viewCache,
+  executeDecisions,
+  executeCachedViewDecisions,
+  showContainers,
+  logger
+}) => {
+  return ({ viewName, pageWideScopeDecisions, formBasedComposedDecisions }) => {
+    if (viewName) {
+      return viewCache.getView(viewName).then(currentViewDecisions => {
+        executeDecisions(pageWideScopeDecisions);
+        executeCachedViewDecisions({
+          viewName,
+          viewDecisions: currentViewDecisions
+        });
+        showContainers();
+
+        return {
+          get decisions() {
+            logger.warn("Decisions property is deprecated");
```

Comment:
Awesome. Should we add: `. Please use "propositions" instead.` ?

Reply:
I like that idea. Let's use complete sentences as well, please. Something like `The decisions property is deprecated. Please use the propositions property instead.`

Reply:
Also, let's de-dupe this string into a shared constant as well, since it's used in a three different places and doesn't minify.

Reply:
ok, will address the changes